docs/conf.py

```diff
@@ -40,11 +40,11 @@
     "sphinx.ext.napoleon",
     "sphinx.ext.autodoc",
     "sphinx.ext.autosummary",
-    "sphinx.ext.mathjax",
     "sphinx.ext.intersphinx",
     "sphinx.ext.doctest",
     "matplotlib.sphinxext.plot_directive",
     "reno.sphinxext",
+    "sphinxcontrib.katex",
```

Comment:
This is a good idea, since the separate docs build uses KaTeX now - best to keep us a bit more in sync.

Reply:
Oh, I actually was just testing it out and included here by accident ... I was going to do it in a separate PR. But it worked fine in my tests.